shared/src/main/scala/pl/metastack/metatime/Range.scala

```diff
@@ -1,5 +1,4 @@
 package pl.metastack.metatime
 
-trait Range {
-
+case class Range (lowerBound: Long, upperBound: Long) {
```

Comment:
You can remove the curly braces. Also remove the space before the left parenthesis.